templates/.erb-lint.yml

```
@@ -1,20 +1,26 @@
 ---
-glob: "**/app/views/**/*.{html,text,js}{+*,}.erb"
 EnableDefaultLinters: true
+exclude:
+  - '**/rails_admin/*'
+  - "vendor/**/*"
 linters:
+  SpaceInHtmlTag:
+    enabled: false
```

```suggestion
    enabled: false
    NoJavascriptTagHelper:
    	enabled: true
	correction_style: 'plain'
```

https://github.com/Shopify/erb_lint/tree/main?tab=readme-ov-file#nojavascripttaghelper

---

I am not sure that I understand this correctly.

---

the indentation is a bit off. I'd add the `NoJavascriptTagHelper` with `correction_style: 'plain'`. the default adds  //<![CDATA[ and //]]> markers (see https://github.com/Shopify/erb_lint/tree/main?tab=readme-ov-file#nojavascripttaghelper)